### PR TITLE
Exposes the CensusContextFactory.

### PR DESCRIPTION
**Pull request by sebright**

### CODE REVIEW

File: core/java/com/google/census/Census.java [hunk 1]
```diff
@@ -26,6 +26,11 @@
 
   private static final CensusContext DEFAULT = CONTEXT_FACTORY.getCurrent();
 
+  /** Returns the default {@link CensusContextFactory}. */
+  public static CensusContextFactory getFactory() {
```
Comment (dinooliva) [inline core/java/com/google/census/Census.java]: Consider naming getContextFactory() or, possibly, getCensusContextFactory().

Comment (sebright) [inline core/java/com/google/census/Census.java]: Done

**Comment by zhangkun83** [comment]
LGTM

**Comment by dinooliva** [comment]
LGTM

**Comment by sebright** [comment]
Thanks!

**Pull request merged**